src/fiatconnect/kyc/KycExpired.test.tsx

```diff
@@ -1,21 +1,19 @@
-import React from 'react'
+import { FiatAccountType, KycStatus as FiatConnectKycStatus } from '@fiatconnect/fiatconnect-types'
```

Comment:
you may want to update your IDE settings to avoid this noisy reordering of imports. in some cases it is even possible to cause bugs by reordering imports, if imports have side effects (which is a mega anti pattern, but still crops up from time to time)